### DISCUSSION TITLE
[질문] 문자열 변수들에 대한 질문

### DISCUSSION THREAD

**Discussion by boogi-woogi** [TEAM-DAME/Dame-Android#79] [2. Q&A] [answered]
![image](https://user-images.githubusercontent.com/88770426/202491402-3af2873a-b81a-4ead-8f70-02a8e15a9a05.png)
이렇게 문자열 변수들이 존재하는 경우 companion object 안에 정의 해놓아야 하는지 아니면 해당 문자열들이 사용되는 모듈 안의 string resource로써 정의해놓아야 하는지 어떠한 방식으로 처리해야하는지 잘 모르겠습니다!

**Comment by onseok** [comment, marked as answer]
문자열 값들은 `shared 모듈 내 src/main/res/strings.xml`에 화면별로 정의해서 따로 관리하고 있습니다.
또한 위 사진 처럼 하드코딩하는 방식보다는 `BindingAdapter(core-ui 모듈에 정의되어 있음)`와 `ViewModel, DataBinding`을 사용하시면 더 좋은 코드가 될 것 같습니다.

**Reply by boogi-woogi** [reply]
추후 리팩토링할때 필수적으로 반영하겠습니다!
